packages/hub/indexers.js

```diff
@@ -55,6 +56,7 @@ class Indexers {
     this._realTimeQueue = [];
     this._seedSchemaMemo = null;
     this._schemaCache = null;
+    this.events = new EventEmitter();
```

Comment:
Is there a reason we aren't just extending Indexers from EventEmitter, so that it would be `indexers.on('index_update', ...)` instead of `indexers.events.on('index_update', ...)`?

Reply:
good idea. I'll give that a shot